### ISSUE TITLE
ML1: window.location.href = ... throws error

### ISSUE THREAD

**Issue by machenmusik** [webmixedreality/exokit#778]

**Comment by avaer** [comment]
This sounds like it’s probably not ml-related; it’s just href handling.

The worker threads pr (https://github.com/webmixedreality/exokit/pull/760) completely changes the notion of a user window so it might make sense to fix over there.

**Comment by machenmusik** [comment]
Is there some other way that works?  ML1 virtual keyboard shift doesn't stick so I can't type colon for port number... so this was a workaround.  window.location.assign() didn't work either

**Comment by modulesio** [comment]
No, but this seems like a bug that's not ML related. Additionally, there is a PR for the shift key functionality.

**Comment by machenmusik** [comment]
(Removed ML1 from bug title, given the discussion above.)
